<launch>

  <node name="tf_moveit_goalsetNode" pkg="coborg_move" type="tf_moveit_goalsetNode" respawn="false" output="screen">
  <param name="originState" value="camera_link"/>
  <param name="manipulation_state" value="standby"/>
  </node>



</launch>